src/api/app/models/flag.rb

```diff
@@ -79,6 +79,7 @@ def default_status
       return all_flag.status if all_flag
     end
     if main_object.kind_of? Package
+      all_flag = main_object.project.flags.where("flag = ? AND repo IS NULL AND architecture_id IS NULL", flag).first
```

Comment:
Please add a description to the commit message explaining why this change was done